content/en/docs/reference/command-line-tools-reference/feature-gates/StorageCapacityScoring.md

```diff
@@ -0,0 +1,23 @@
+---
+title: StorageCapacityScoring
+content_type: feature_gate
+_build:
+  list: never
+  render: false
+
+stages:
+  - stage: alpha
+    defaultValue: false
+    fromVersion: "1.33"
+---
+This feature adds a scoring method for pod scheduling with
+[the topology-aware volume provisioning](/blog/2018/10/11/topology-aware-volume-provisioning-in-kubernetes/).
+This feature eases to schedule pods on nodes with either the most or least
```

Comment:
This sentence could use some help from native speakers.

Reply:
Unfortunately, I don't have any native English speakers near me...